src/utils/Constants.ts

```diff
@@ -91,7 +91,8 @@ export class Constants {
 		LAST_NAME: "last name",
 		LINK_TO_FILE: "link_to_file",
 		CHOSEN_PHOTO_ID: "chosen photo ID",
-
 	};
+
+	static readonly TXMA_FIELDS_TO_SHOW = ["event_name", "documentType"];
```

Comment:
@hemavana @Caitlin-cooling Let me know what fields in F2F we'd be keen to see here please to help with debugging

Reply:
I know we will already have `session_id` and `govuk_signin_journey_id` appended but I think not obfuscating it might be helpful too? Otherwise LGTM most of it is PII

Reply:
Done